test/fork/Fork.t.sol

```diff
@@ -12,8 +12,8 @@ abstract contract Fork_Test is Base_Test {
                                      CONSTANTS
     //////////////////////////////////////////////////////////////////////////*/
 
-    IERC20 internal immutable asset;
-    address internal immutable holder;
+    IERC20 internal asset;
+    address internal holder;
```

Comment:
Why have you removed `immutable` here? Let's make them uppercase

Reply:
It was easier to remove the state, it is not relevant here to use `immutable` but as you wish.

Reply:
It is relevant to have these variables remain immutable - the goal is to prevent accidental modifications.